.github/workflows/cli_core_e2e_test.yaml

```
@@ -2,9 +2,9 @@ name: Tanzu CLI Core Tests
 
 on:
   pull_request:
-    branches: [main, release-*]
-    paths:
-      - "*"
+    branches: [main]
+  push:
+    branches: [main, test*]
```

What is the reason behind using `test-*` here?
```suggestion
    branches: [main, release-*]
```

---

updated.